app.js

```diff
@@ -0,0 +1,152 @@
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/watermarker.png',
+    id: 'forth-container',
+    idBtn: 'notsee3',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Profesional Art Printing Data More',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/Blurwatermark.png',
+    id: 'fifth-container',
+    idBtn: 'notsee4',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Data Dashboard Healthcare',
+
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/3watermark.png',
+    id: 'sixth-container',
+    idBtn: 'notsee5',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Website Protfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+
+];
+
+function renderCards(i) {
+  const div = document.createElement('div');
+  div.className = 'middle-1-container1';
+
+  div.innerHTML = `<div id="${projects[i].id}" class="middle-1-container1">
+        <h2 class="mobile">${projects[i].name}</h2>
+      <h3 class="desktop">${projects[i].name1}</h3>
+        
+        
+        <p>${projects[i].description}</p>
+  
+        <ul class="cardbtn">
+              ${projects[i].technologies.map((tech) => `<li><button type="button" id="${projects[i].idSmallbtn}" class="middle-1-container1-btn">${tech}</button></li>`).join('')}
+        </ul>
+        <button type="button" id="${projects[i].idBtn}" onclick="openPopup()" class="see-button-one1">See project</button>
+      </div>`;
+  // eslint-disable-next-line prefer-const
+  let nameid = projects[i].id;
+
+  // eslint-disable-next-line prefer-template
+  div.querySelector('#' + nameid).style.backgroundImage = `url("${projects[i].url}")`;
+  document.querySelector('.middle-section').appendChild(div);
+}
+
+// eslint-disable-next-line no-unused-vars
+function openPopup() {
+  const pop = document.querySelector('.popup2');
+  // eslint-disable-next-line quotes
+  pop.style.display = "flex";
+  // eslint-disable-next-line no-var
+  const blur = document.getElementById('middlesec');
+  blur.classList.toggle('active');
+}
+
+// eslint-disable-next-line no-unused-vars
+function closePopup() {
+  const pop = document.querySelector('.popup2');
+  pop.style.display = 'none';
+}
+
+renderFirstCard();
+// eslint-disable-next-line no-plusplus
+for (let i = 0; i < 6; i++) {
+  renderCards(i);
+}
```

Comment:
- [ ] **Required:** You are required to validate the email string against Uppercase being stored but this isn't done. I have not also seen in your codebase where you tried to implement this. Kindly implement.

![Not validated](https://user-images.githubusercontent.com/27709832/163488389-9753ad09-47b8-42af-953a-66356f1a8399.gif)

- [ ] **Required:** Fix console error.

![image](https://user-images.githubusercontent.com/27709832/163488756-28c59736-be8b-4668-b546-d005306a37f7.png)

- [ ] **Required:** You are also required to show an error message near the submit button for your form if the string isn't valid.